### PR TITLE
Complete Awesome books project 1

### PR DESCRIPTION
**Pull request by coderacheal**
### **Complete all project requirements for the awesome books project**

- [ ] Create a collection that keeps a list of books
- [ ] Create a function to add a new book to the collection, with title and author.
- [ ] Create a function to remove a book from the collection
- [ ] Display all books saved in the collection at the top part of the page.
- [ ] When a user clicks the "Add" button:
    
      -    A new book is added to the collection.
      -    The new book is displayed in the page.

- [ ] When a user clicks the "Remove" button:

      - The correct book is removed from the collection.
      - The correct book disappears from the page.

- [ ] The data is preserved in the browser's memory by using [localStorage]

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,43 +1,48 @@
...
-    }
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
 const bookArchive = [];
-const form  = document.querySelector('.form');
+const form = document.querySelector('.form');
```
Comment (mukhammadrizooff) [inline script.js]: - [ ] [REQUIRED] Good job so far! Please It should have a function that `removes` a book from the collection (hint: You should use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method) 🙏🏼 
- `[HINT]`
```javascript
const books = [];

const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```
#
image
#

File: script.js [hunk 2]
```diff
@@ -1,43 +1,48 @@
...
-    <p class="the-auhtor">${book.author}</p>
-    <button class='removeBook' >Remove</button>
-    `
-}
+  bookArchive.push(book);
+  addNewBook(book);
 
-// const removeButton = document.querySelector('.removeBook');
-// console.log(removeButton)
+  localStorage.setItem('bookInfo', JSON.stringify(bookArchive));
```
Comment (mukhammadrizooff) [inline script.js]: - [ ] [REQUIRED] Good job so far! Data should be preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reloading the page manually to check this after `adding` a book and `removing` a book. Date is being saved to local storage but it is not rendering to display, please fix it 🙏🏼 

ezgif-4-831aa3158c

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @coderacheal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @coderacheal 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**